funcx_endpoint/funcx_endpoint/endpoint/endpoint_manager.py

```diff
@@ -164,8 +163,11 @@ def start_endpoint(self, name, endpoint_uuid, endpoint_config):
         if not endpoint_config.config.executors:
             raise Exception(f"Endpoint config file at {endpoint_dir} is missing executor definitions")
 
-        funcx_client = FuncXClient(funcx_service_address=endpoint_config.config.funcx_service_address,
-                                   check_endpoint_version=True)
+        funcx_client_options = {
```

Comment:
This is okay for now, but ideally we want this to be end-user configurable for any arbitrary client options.